packages/parchment/lib/src/codecs/markdown.dart

```diff
@@ -486,8 +498,11 @@ class _ParchmentMarkdownEncoder extends Converter<ParchmentDocument, String> {
     return buffer.toString();
   }
 
-  void _writeAttribute(StringBuffer buffer, ParchmentAttribute? attribute,
-      {bool close = false}) {
+  void _writeAttribute(
```

Comment:
We'd rather avoid trailing commas in function declarations when not a Flutter widget